_inc/client/at-a-glance/plugins.jsx

```diff
@@ -46,19 +50,30 @@ const DashPluginUpdates = React.createClass( {
 
 		if ( 'updates-available' === pluginUpdates.code ) {
 			return(
-				<DashItem label={ labelName } status="is-warning">
+				<DashItem
+					label={ labelName }
+					module="manage"
+					status="is-warning"
+				>
+					<h2 className="jp-dash-item__count">
+						{
+							__( '%(number)s plugin', '%(number)s plugins', {
+								count: pluginUpdates.count,
+								args: {
+									number: pluginUpdates.count
+								}
+							} )
+						}
+					</h2>
 					<p className="jp-dash-item__description">
-						<strong>
-							{
-								__( '%(number)s plugin needs updating.', '%(number)s plugins need updating.', {
-									count: pluginUpdates.count,
-									args: {
-										number: pluginUpdates.count
-									}
-								} )
-							}
-						</strong>
-						<br/>
+						{
```

Comment:
@dereksmart When you get a minute, can you check to make sure I did this section right? (I don't fully understand it, but I guessed based on your work and it works)